hadoop-ozone/ozonefs-common/src/main/java/org/apache/hadoop/fs/ozone/BasicOzoneClientAdapterImpl.java

```diff
@@ -668,12 +668,22 @@ public void deleteSnapshot(String pathStr, String snapshotName)
   public SnapshotDiffReport getSnapshotDiffReport(Path snapshotDir,
       String fromSnapshot, String toSnapshot)
       throws IOException, InterruptedException {
-    boolean takeTemporarySnapshot = false;
-    if (toSnapshot.equals(ACTIVE_FS_SNAPSHOT_NAME)) {
-      takeTemporarySnapshot = true;
+    boolean takeTemporaryToSnapshot = false;
```

Comment:
should not allow if both fromSnapshot and toSnapshot is empty

Reply:
that is a valid scenario in HDFS atleast, it is same as computing diff two same snapshots. same as computing diff b/w snapshot s1 & s1, if that is allowed, then this should be allowed as well. Empty Snapshot is just a special case which represents the current state & is same as any other snapshot